scripts/codechecks/hqDefine.sh

```diff
@@ -0,0 +1,22 @@
+# run as
+# bash scripts/codechecks/hqDefine.sh
```

Comment:
good point

On Wed, Apr 20, 2016 at 4:53 PM, Ethan Soergel <notifications@github.com>
wrote:

> In scripts/codechecks/hqDefine.sh
> <https://github.com/dimagi/commcare-hq/pull/11354#discussion_r60485404>:
>
> > @@ -0,0 +1,22 @@
> > +# run as
> > +# bash scripts/codechecks/hqDefine.sh
>
> just curious - why no #!/bin/bash and chmod +x
> scripts/codechecks/hqDefine.sh?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/11354/files/65a73fe8f5b5131102a7bc4f010f084edfaf7235#r60485404>
>